impl/src/settings.rs

```diff
@@ -369,6 +387,51 @@ fn validate_settings(settings: &mut RazeSettings) -> Result<(), RazeError> {
   Ok(())
 }
 
+/** Parses raze settings from the contents of a `Cargo.toml` file */
+fn parse_raze_settings(toml_contents: CargoToml) -> Result<RazeSettings, RazeError> {
+  // Workspace takes precidence
+  if let Some(raze) = toml_contents
+    .workspace
+    .and_then(|pkg| pkg.metadata.and_then(|data| data.raze))
```

Comment:
This appears to add a third possible configuration location:

- [raze]
- [package.metadata.raze]
- [workspace.metadata.raze]

This was not mentioned in the PR description. Why is this being added?

Reply:
This PR was opened before https://github.com/google/cargo-raze/pull/274 was merged and I had to make this change after a rebase. I just slipped my mind to add it to the PR description. This is being added because it makes sense to me when authoring projects using cargo workspaces. Associating raze settings with a package gets confusing when there's no root package. To solve for this case, we should use `[workspace.metadata]`

Reply:
I also updated the PR description.